packages/services/f-feature-management/src/logger.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Default logger implementation.  The interface matches that used by various Fozzie components.
+ */
+const logger = {
+    logError (logMessage, store, logPayload = {}) {
```

Comment:
Just a reminder when you productionise this bit; pass a property to the payload (or add in these methods)
```suggestion
    logError (logMessage, store, logPayload = {}) {
      logPayload = {
        ...logPayload,
        tags: 'FeatureManagement'
      };
```

Reply:
Shall I just commit your suggestion then?

Reply:
Nah just keep it in mind for now; that was all